src/accounts/views.py

```diff
@@ -57,6 +60,17 @@ def create_profile(user=None, **kwargs):
                                                     xii_percentage=kwargs['xii_percentage'], )
         userprofile.save()
 
+        # Sends email to the registered student
```

Comment:
Put all this logic in a function and call it instead. You should probably put this function in the model itself, making to easier to be called again if needed from other parts of the code